### PR TITLE
fix(script): fix release build fails of current nightly rust

### PR DESCRIPTION
**Pull request by ailisp**
After update rust rust to 2020-05-15 nightly, binary except for neard and vm-runner-standalone failed with error message:
```
nearcore/target/release/deps/sysinfo-4b1af5e28b5a43b6.sysinfo.8i5nip17-cgu.0.rcgu.o: file not recognized: File format not recognized
```
It's reproducible locally by `cargo build -p state-viewer`. Fortunately, neard and vm-runner-standalone release build is fine and this two is we really care of performance. So before next rust nightly upgrade I debug build this failed binaries. (Note: https://rust-lang.github.io/rustup-components-history/ there isn't a newer rustc that has rls so I'll wait)

Test Plan
-----------
Change buildkite ci pipeline to also run binary/docker build on this PR and see it pass

### CODE REVIEW

**Comment by ailisp** [comment]
@bowenwang1996 
> have you tried other versions of nightly? Maybe it can be solved by choosing a different version.

There isn't newer rust with rls and I don't want to downgrade rust, we'll need to wait 2020-06-01 I guess

**Comment by bowenwang1996** [comment]
@ailisp I think it's fine. We can wait a bit and use temporary work around like this without merging it into master.

File: chain/client/tests/cross_shard_tx.rs [hunk 1]
```diff
@@ -420,9 +420,9 @@ mod tests {
                 block_production_time,
                 drop_chunks,
                 !test_doomslug,
-                20,
+                5,
                 test_doomslug,
-                vec![true; validators.iter().map(|x| x.len()).sum()],
+                vec![false; validators.iter().map(|x| x.len()).sum()],
```
Comment (bowenwang1996) [inline chain/client/tests/cross_shard_tx.rs:425]: why this change? It actually changed the semantics of this test.

Comment (ailisp) [reply chain/client/tests/cross_shard_tx.rs]: it is a different change, revert it
